src/measurement/vcpu_types.rs

```diff
@@ -0,0 +1,160 @@
+// SPDX-License-Identifier: Apache-2.0
+
+//! Enum of different exisitng AMD EPYC VCPUs
+use std::{convert::TryFrom, fmt, str::FromStr};
+
+use crate::error::MeasurementError;
+
+/// All currently available QEMU VCPU types
+#[derive(Debug, Clone, Copy, PartialEq)]
+pub enum CpuType {
+    /// EPYC
+    Epyc = 0,
```

Comment:
I don't think you need to explicitly say 0, 1, 2, 3, etc, as we are not skipping anything. Those values  will be inferred.